charts/thoras/templates/monitor/deployment.yaml

```diff
@@ -75,10 +75,10 @@ spec:
               secretKeyRef:
                 name: thoras-slack
                 key: webhookID
-          - name: SLACK_WORKSPACE_ID
```

Comment:
Do we need to worry about being backwards compatible? Seems we wouldn't want to release this until what's https://github.com/thoras-ai/platform/pull/73 is merged to `main`

Reply:
Yep, 100%. This would be a breaking change -- we'll just need to include instructions that explain the new value layout when we ask customers to upgrade. We'll also want to bump the major version of the chart to `3.x.x` since we're breaking (versions are cheap tho!!)

It may not always be preferable to do a hard break, but I think it makes sense for this for now

I've filed a [ticket](https://github.com/thoras-ai/platform/issues/85) to track this

Reply:
I bumped the version to 4.0.1